tests/helics/application_api/test_files/example_combo_fed.json

```diff
@@ -45,8 +45,9 @@
 },
 {
 "key":"pub2", // the name of the publication
- "type":"double" // the type assocaited with a publication (optional)
+ "type":"double", // the type assocaited with a publication (optional)
  //no global:true implies this will have the federate name prepended like comboFed/pub2
+ "info":{"field1":45,"field2":99}
```

Comment:
While I also prefer the "pass in real JSON" on principle, I'm concerned about the issues this might cause for the time being, do we want to try to support this functionality immediately? My impression was we were going to attempt to enforce string-format for now.
something like:
`"info":"{\"field1\":45,\"field2\":99}"`

Reply:
There seemed to be a way of doing it with minimal code in the call to getValueOrDefault with json so I just added it.    What issues are you concerned with?

Reply:
As long as it doesn't cause encoding issues, I suppose it's a non-issue. My only real concern was that we may see a lot of "I gave it JSON but got an escaped string back and had to add a JSON decoder on my end to read it" which is the behavior I would expect, but might not be intuitive, or something of that nature.

Reply:
I think we will just have to make sure we document what it does and doesn't do properly.  If it was a json structure to begin with it should come out as a json interpretable string.    In C++ you can probably just stick whatever data you wanted in there since std::string isn't limited to text characters.  That isn't going to work on C interfaces though